documentation/modules/ROOT/pages/development/engine.adoc

```diff
@@ -20,6 +20,19 @@ They still want the same data change events, but prefer to have the connectors s
 
 This `debezium-api` module defines a small API that allows an application to easily configure and run {prodname} connectors using {prodname} Engine.
 
+Beginning with the 2.6.0 release, {prodname} provides two implementations of the `DebeziumEngine` interface.
+The older `EmbeddedEngine` implementation runs a single connector that uses only one task.
+The connector emits all records sequentially.
+ 
+This the default implementation.
+Beginning with the 2.6.0 release, a new `AsyncEmbeddedEngine` implementation is available.
```

Comment:
```suggestion
The connector emits all records sequentially.
 This the default implementation.
 
Beginning with the 2.6.0 release, a new `AsyncEmbeddedEngine` implementation is available.
```
Probably should remove the blank line on L26, and then start the new paragraph after that by adding a blank line after the sentence that begins _This is the default ..._